### PR TITLE
UICIRC-848: Create a new permission Settings (Circulation): Can view staff slips

### PR DESCRIPTION
**Pull request by Dmitriy-Litvinenko**
## Purpose
Create a new permission Settings (Circulation): Can view staff slips

# Refs
https://issues.folio.org/browse/UICIRC-848

### CODE REVIEW

**Review by artem-blazhko — APPROVED** [review]

File: src/settings/StaffSlips/StaffSlipForm.js [hunk 1]
```diff
@@ -48,8 +48,7 @@ const StaffSlipForm = (props) => {
     : formatMessage({ id: 'ui-circulation.settings.staffSlips.new' });
 
   const footerPaneProps = {
-    pristine,
-    submitting,
+    disabled: pristine || submitting || !stripes.hasPerm('ui-circulation.settings.edit-staff-slips'),
```
Comment (Dmytro-Melnyshyn) [inline src/settings/StaffSlips/StaffSlipForm.js:51]: FOLIO's approach is to not show an element if a user doesn't have permission.

File: package.json [hunk 2]
```diff
@@ -192,27 +192,33 @@
           "ui-circulation.settings.edit-circulation-rules"
         ]
       },
+      {
+        "permissionName": "ui-circulation.settings.view-staff-slips",
```
Comment (mariia-aloshyna) [inline package.json:196]: Add this new perm name to translations also, plase

Comment (Dmitriy-Litvinenko) [reply package.json:196]: Thank you, fixed (https://github.com/folio-org/ui-circulation/pull/1127/commits/67792109215c963a8d8bc2b6a7e71ebb612619a3) Could you please rereview?

File: src/settings/StaffSlips/StaffSlipForm.js (hunk 1 above)
Comment (Dmitriy-Litvinenko) [reply src/settings/StaffSlips/StaffSlipForm.js]: Thank you, fixed (https://github.com/folio-org/ui-circulation/pull/1127/commits/ed0c3a77de8139e2c29216ec1b77a4a44e497d06) Could you please re-review?

**Review by Dmytro-Melnyshyn — APPROVED** [review]

**Pull request merged**